Camera: {Focal: 13.971313932573906, High: 650.0}
Center: {Camera_high: 670, Center_X: 317, Center_Y: 251, Front: 88, Horizon: 189,
  Inner: 46, Outer: 279, X: 308, Y: 266}
FPS: 20
HSV:
  Ball: [0, 102, 55, 100, 21, 100]
  Blue: [170, 258, 64, 81, 36, 67]
  Green: [113, 208, 51, 70, 14, 33]
  White: [0, 360, 0, 27, 0, 31]
  Yellow: [30, 92, 28, 98, 24, 100]
  black: {angle: 3, gray: 35}
  white: {angle: 33, gray: 170}
Parameterbutton: 4
SCAN: {Angle_Near_Gap: 4, Angle_range_1: 2, Angle_range_2_3: 4, Dont_Search_Angle_1: 47,
  Dont_Search_Angle_2: 164, Dont_Search_Angle_3: 285, Magn_Far_End: 212, Magn_Far_Start: 141,
  Magn_Middle_Start: 93, Magn_Near_Gap: 4, Magn_Near_Start: 40}
blackItem: {angle: 3, gray: 40}
whiteline: {angle: 4, gray: 100}
